src/tier-types.ts

```diff
@@ -522,6 +522,10 @@ export interface CheckoutParams {
   features?: Features | Features[]
   trialDays?: number
   requireBillingAddress?: boolean
+  tax?: {
+    automatic?: boolean
+    collectId?: boolean
```

Comment:
Any reason this and `collect_id` on `tax` in both CheckoutRequest and CheckoutParams be `collect_id` instead of one being `collectId`? If they can both be the same name, it would be nice to have a single type and use for both fields. In Go there is the Taxation type, so to keep up with the Go reference SDK, we can name it Taxation, if needed.

Reply:
Oh I thought maybe there was a convention to prefer camel case for params based on other types in the file. I will update it!

Reply:
Oh no. You're right. Let's keep it camelCase and be consistent with the rest of the API types.

Reply:
Reverted back to previous style. @bmizerany